src/styles/common.less

```diff
@@ -304,7 +304,7 @@ p {
 .menu-tab-content {
   margin: 2em;
   .header {
-    margin: 24px 0;
+    margin: 0px 0px 24px 0px;
```

Comment:
This gets the header closer to the tabs, rather than having a bit of a gap, but it also puts the header text and the search bar on the Packages tab on different heights creating a bit of a disconnect in the UI.

We can apply a similar change to `.header-row > .search-group`, or apply it to the full `.header-row` and ensure other headers are wrapped in a `header-row` on other pages.

Additionally we could use `margin-bottom: 24px` instead depending on how we feel about using the specific property versus margin with multiple values.

Reply:
used the margin-bottom suggestion and applied it to the search block as well